framework/src/modules/chain/transport/transport.js

```diff
@@ -215,37 +215,32 @@ class Transport {
 	 */
 
 	/**
-	 * Description of blocks.
-	 *
-	 * @todo Add @param tags
-	 * @todo Add description of the function
+	 * Returns a set of full blocks starting from the ID defined in the payload up to
+	 * the current tip of the chain.
+	 * @param {object} payload
+	 * @param {string} payload.blockId - The ID of the starting block
+	 * @return {Promise<*|Promise<*>>}
```

Comment:
What are the possible returns? `this.blocksModule.loadBlocksFromLastBlockId(payload.blockId, 34);` ?